pages/index.qmd

```diff
@@ -3,23 +3,21 @@ title: "Introduction"
 ---
 
 
+
+This is a dashboard for the [COVID-19 Forecast Hub](https://github.com/CDCgov/covid19-forecast-hub) that is organized by the US CDC. A page with forecast evaluations will be added at a later date.
```

Comment:
I suggest pointing to the prior COVID-19 Forecast Hub page with an explainer about the new hub, and the launch date, etc... If we don't solve this in this PR, then let's file an issue to make this change.